internal/cmd/config.go

```diff
@@ -317,15 +322,17 @@ func newConfig(options ...configOption) (*Config, error) {
 		return nil, err
 	}
 
-	bds, err := xdg.NewBaseDirectorySpecification()
-	if err != nil {
-		return nil, err
-	}
+	xdgConfigHome := xdg.ConfigHome
+	xdgConfigDirs := append([]string{xdgConfigHome}, xdg.ConfigDirs...)
+	xdgDataHome := xdg.DataHome
+	xdgDataDirs := append([]string{xdgDataHome}, xdg.DataDirs...)
+	xdgCacheHome := xdg.CacheHome
+	xdgRuntimeDir := xdg.RuntimeDir
 
 	logger := zerolog.Nop()
 
 	c := &Config{
-		ConfigFile: newConfigFile(bds),
+		ConfigFile: newConfigFile(xdgCacheHome),
```

Comment:
This should be `xdgConfigHome`.

Reply:
The `bds` argument that that function took prior was only used to determine the cache directory, and as such I simply passed the required cache path directly. I realise that this looks pretty confusing, though.